bigquery/api/async_query.py

```diff
@@ -103,7 +136,8 @@ def main(project_id, query_string, batch, num_retries, interval):
 # [START main]
 if __name__ == '__main__':
     parser = argparse.ArgumentParser(
-        description='Loads data into BigQuery.')
+        description=__doc__,
```

Comment:
Ah this is a nice idea.

On the other hand, the README.md files in the sub directories are almost empty, although we say "For more detailed introduction to a product, check the README in the corresponding folder." in the top level README.

I would rather keep the following parts in the docstring 

```
Command-line application to perform an asynchronous query in BigQuery.

This sample is used on this page:

    https://cloud.google.com/bigquery/querying-data#asyncqueries
```

and move the rest to the READMEs in the subdirectories?